### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#273] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131757](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d67498e4-4c00-4030-91af-0da196312f78)
![Screenshot 2023-11-28 131809](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a73d6150-0846-4a31-8a26-718409969974)

**Comment by Elia-Belli** [comment]
- Espressione di $L_A$:

$$L_A\begin{pmatrix}x_1\\
x_2\\
x_3\end{pmatrix}=\begin{pmatrix}x_1+3x_2-x_3\\
2x_1+x_2-x_3\\
2x_1-x_2\end{pmatrix}$$

- Immagine del vettore $(1,2,1)$ tramite $L_A$

$$L_A\begin{pmatrix}1\\
2\\
1\end{pmatrix}=\begin{pmatrix}6\\
3\\
0\end{pmatrix}$$

- Immagine dei vettori della base canonica tramite $L_A$:

$$L_A\begin{pmatrix}1\\
0\\
0\end{pmatrix}=\begin{pmatrix}1\\
2\\
2\end{pmatrix};L_A\begin{pmatrix}0\\
1\\
0\end{pmatrix}=\begin{pmatrix}3\\
1\\
-1\end{pmatrix};L_A\begin{pmatrix}0\\
0\\
1\end{pmatrix}=\begin{pmatrix}-1\\
-1\\
0\end{pmatrix}$$

- $L_A$ iniettiva $\iff Ker\ L_A =\\{\underline 0\\}$, dobbiamo quindi risolvere il sistema omogeneo:

$$\begin{cases}x_1+3x_2-x_3=0\\
2x_1+x_2-x_3=0\\
2x_1-x_2=0\end{cases}\rightarrow\begin{pmatrix}1 & 3 &-1\\
0 & -5 & 1\\
0 & 0 & -2/5\end{pmatrix}$$

Che ha infatti soluzione unica $\underline 0\Rightarrow L_A$ iniettiva

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5b615313-8e65-41d1-a7a6-4b29c177bb15)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/14372bf8-361d-4dc4-956c-6e9b609d17ea)
